<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>687</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Escutor Digital</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/images/images/pngtree-vector-drawing-brush-icon-png-image_313055.jpg</normaloff>:/images/images/pngtree-vector-drawing-brush-icon-png-image_313055.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Raio da Elipse</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber_5"/>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSliderRY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumber_6"/>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSliderRZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSliderRX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Ry</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_7"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Rz</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Rx</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="ColorWindow" name="widgetColor" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Cor</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber_9"/>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumber_10"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSliderBlue">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="lcdNumber_12"/>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_11"/>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSliderGreen">
       <property name="sliderPosition">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSliderRed">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSliderAlpha">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Tamanho do Cubo</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSliderY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSliderX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSliderZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_3"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Raio da Espera</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber_4"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSliderRD">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Plotter" name="widgetPlotter" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>621</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>471</width>
      <height>431</height>
     </size>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>-20</y>
       <width>162</width>
       <height>426</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8"/>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>66</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">t</string>
       </property>
       <property name="text">
        <string>Eixo X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlidereixo">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_8"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewBox"/>
   <addaction name="actionPutVoxel"/>
   <addaction name="actionCutVoxel"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionCutBox"/>
   <addaction name="actionPutSphere"/>
   <addaction name="actionCutSphere"/>
   <addaction name="actionPutEllipsoid"/>
   <addaction name="actionCutEllipsoid"/>
   <addaction name="actionSave"/>
   <addaction name="actionSair"/>
  </widget>
  <action name="actionPutVoxel">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/pngtree-vector-drawing-brush-icon-png-image_313055.jpg</normaloff>
     <normalon>:/images/images/index2.jpeg</normalon>:/images/images/pngtree-vector-drawing-brush-icon-png-image_313055.jpg</iconset>
   </property>
   <property name="text">
    <string>Put Voxel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCutVoxel">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/images.png</normaloff>:/images/images/images.png</iconset>
   </property>
   <property name="text">
    <string>cutVoxel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPutBox">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/putbox.png</normaloff>:/images/images/putbox.png</iconset>
   </property>
   <property name="text">
    <string>putBox</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCutBox">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/84583.png</normaloff>:/images/images/84583.png</iconset>
   </property>
   <property name="text">
    <string>cutBox</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPutSphere">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/blue-icon-hi.png</normaloff>:/images/images/blue-icon-hi.png</iconset>
   </property>
   <property name="text">
    <string>putSphere</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCutSphere">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/cutesfera.png</normaloff>:/images/images/cutesfera.png</iconset>
   </property>
   <property name="text">
    <string>cutSphere</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPutEllipsoid">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/elipse-512.png</normaloff>:/images/images/elipse-512.png</iconset>
   </property>
   <property name="text">
    <string>putEllipsoid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCutEllipsoid">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/300px-Ellipse_black.svg.png</normaloff>:/images/images/300px-Ellipse_black.svg.png</iconset>
   </property>
   <property name="text">
    <string>cutEllipsoid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionNewBox">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/Files-New-File-icon.png</normaloff>:/images/images/Files-New-File-icon.png</iconset>
   </property>
   <property name="text">
    <string>NewBox</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionShow_OFF">
   <property name="text">
    <string>OFF</string>
   </property>
   <property name="toolTip">
    <string>OFF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/save-icon-4.png</normaloff>:/images/images/save-icon-4.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/images/images/com.pandaz.app.forcequit.png</normaloff>:/images/images/com.pandaz.app.forcequit.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <signal>moveX(int)</signal>
    <signal>moveY(int)</signal>
    <signal>planeChosen(int)</signal>
    <slot>mudaFrequencia(int)</slot>
    <slot>changeRed(int)</slot>
    <slot>changeGreen(int)</slot>
    <slot>changeBlue(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ColorWindow</class>
   <extends>QWidget</extends>
   <header>colorwindow.h</header>
   <container>1</container>
   <slots>
    <slot>changeRed()</slot>
    <slot>changeGreen()</slot>
    <slot>changeBlue()</slot>
    <slot>changeAlpha()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRD</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_12</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>645</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>664</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlidereixo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
